react/components/AccordionFilterItem.js

```diff
@@ -56,6 +62,19 @@ const AccordionFilterItem = ({
     }
   }
 
+  const applyAccordionCollapseClasses = (param) => {
+    switch(param) {
+      case "Button":
+        return !isCollapsed ? `${handles.accordionCollapseButtonOpen}` : `${handles.accordionCollapseButtonClose}`
+      case "Content":
+        return !isCollapsed ? `${handles.accordionCollapseContentOpen}` : `${handles.accordionCollapseContentClose}`
+      case "Container":
+        return !isCollapsed ? `${handles.accordionCollapseContainerOpen}` : `${handles.accordionCollapseContainerClose}`
+      default:
+        return null
+    }
+  }
+
```

Comment:
Could it be just an object instead of a function? Something like this

```javascript
const accordionCollapseClasses = {
  button: !isCollapsed ? `${handles.accordionCollapseButtonOpen}` : `${handles.accordionCollapseButtonClose}`,
  content: !isCollapsed ? `${handles.accordionCollapseContentOpen}` : `${handles.accordionCollapseContentClose}`,
  container: !isCollapsed ? `${handles.accordionCollapseContainerOpen}` : `${handles.accordionCollapseContainerClose}`
}
```

And then

```javascript
<div
  className={`${accordionCollapseClasses.container} pl7`}
></div>
```

Reply:
@hiagolcm 

Thanks for the sugestion!

Correction made.

Could you review it again, please?